src/utils/io/progress_bar.rs

```diff
@@ -81,6 +81,7 @@ impl ProgressBar {
         }
     }
 
+    #[allow(dead_code)]
```

Comment:
Use or delete.

Reply:
Exactly, let's remove all of the untested code.